src/script/assets/AssetCrypto.ts

```diff
@@ -17,6 +17,12 @@
  *
  */
 
+export interface EncryptedAsset {
```

Comment:
Can we use `EncryptedAsset` from [here](https://github.com/wireapp/wire-web-packages/blob/16579f5a41c36769d8e4cd1a502dfae32dbd31b2/packages/core/src/main/cryptography/EncryptedAsset.ts)?

Reply:
Unfortunately not, since we use `ArrayBuffer` instead of `Buffer`.